adapters/audienceNetwork/facebook.go

```diff
@@ -239,102 +232,106 @@ func (this *FacebookAdapter) extractPlacementAndPublisher(out *openrtb.Imp) (str
 		}
 	}
 
-	placementId := fbExt.PlacementId
-	publisherId := fbExt.PublisherId
+	placementID := fbExt.PlacementId
+	publisherID := fbExt.PublisherId
 
 	// Support the legacy path with the caller was expected to pass in just placementId
 	// which was an underscore concantenated string with the publisherId and placementId.
 	// The new path for callers is to pass in the placementId and publisherId independently
 	// and the below code will prefix the placementId that we pass to FAN with the publsiherId
 	// so that we can abstract the implementation details from the caller
-	toks := strings.Split(placementId, "_")
+	toks := strings.Split(placementID, "_")
 	if len(toks) == 1 {
-		if publisherId == "" {
+		if publisherID == "" {
 			return "", "", &errortypes.BadInput{
 				Message: "Missing publisherId param",
 			}
 		}
 
-		return placementId, publisherId, nil
+		return placementID, publisherID, nil
 	} else if len(toks) == 2 {
-		publisherId = toks[0]
-		placementId = toks[1]
+		publisherID = toks[0]
+		placementID = toks[1]
 	} else {
 		return "", "", &errortypes.BadInput{
-			Message: fmt.Sprintf("Invalid placementId param '%s' and publisherId param '%s'", placementId, publisherId),
+			Message: fmt.Sprintf("Invalid placementId param '%s' and publisherId param '%s'", placementID, publisherID),
 		}
 	}
 
-	return placementId, publisherId, nil
+	return placementID, publisherID, nil
 }
 
 // XXX: This entire function is just a hack to get around mxmCherry 11.0.0 limitations, without
 // having to fork the library and maintain our own branch
-func modifyImpCustom(json []byte, imp *openrtb.Imp) ([]byte, error) {
+func modifyImpCustom(jsonData []byte, imp *openrtb.Imp) ([]byte, error) {
 	impType, ok := resolveImpType(imp)
```

Comment:
Removed use of jsonparser.Set and replaced with standard library. The Set and Delete operations in jsonparser are experimental and shouldn't be used.